[2023-09-04T06:02:14.169+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2023-09-04T06:02:14.182+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2023-09-04T06:02:14.183+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:14.200+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-03-10 00:00:00+00:00
[2023-09-04T06:02:14.207+0000] {standard_task_runner.py:57} INFO - Started process 5210 to run task
[2023-09-04T06:02:14.210+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-03-10T00:00:00+00:00', '--job-id', '390', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpznqi1w47']
[2023-09-04T06:02:14.212+0000] {standard_task_runner.py:85} INFO - Job 390: Subtask insert_into_table
[2023-09-04T06:02:14.271+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-10T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:14.376+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-10T00:00:00+00:00'
[2023-09-04T06:02:14.377+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-03-10', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:02:14.387+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:14.395+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:14.408+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-03-10', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:02:14.412+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:02:14.424+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220310T000000, start_date=20230904T060214, end_date=20230904T060214
[2023-09-04T06:02:14.463+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:14.480+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:05:00.336+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2023-09-04T06:05:00.366+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2023-09-04T06:05:00.367+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:05:00.405+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-03-10 00:00:00+00:00
[2023-09-04T06:05:00.420+0000] {standard_task_runner.py:57} INFO - Started process 6539 to run task
[2023-09-04T06:05:00.426+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-03-10T00:00:00+00:00', '--job-id', '940', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpyjpm5b7a']
[2023-09-04T06:05:00.431+0000] {standard_task_runner.py:85} INFO - Job 940: Subtask insert_into_table
[2023-09-04T06:05:00.589+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-10T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:05:00.890+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-10T00:00:00+00:00'
[2023-09-04T06:05:00.892+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-03-10', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:05:00.921+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:00.946+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:00.983+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-03-10', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:05:00.997+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:05:01.056+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220310T000000, start_date=20230904T060500, end_date=20230904T060501
[2023-09-04T06:05:01.131+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:05:01.185+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
